src/redux/categories/categories.js

```diff
@@ -0,0 +1,27 @@
+// Define an action type for checking the status.
+const CHECKSTATS = 'Under construction';
```

Comment:
- [ ] Also here the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.